### ISSUE TITLE
Path of markers are dashed/dotted if the line is dashed/dotted

### ISSUE THREAD

**Issue by Duesterwald** [nschloe/matplotlib2tikz#93]
If you try to use markers with dashed or dotted lines, the dashing/dotting will apply to the marker path as well. This is due to the behaviour of PGFPlots. However, the user will never want that, as this is different from what matplotlib creates.

It can be fixed by simply adding `solid` to the `mark options` of an `\addplot` command. It is possible to do it by hand afterwards, but it would be nice if this happened automatically.

**Comment by nschloe** [comment]
Good catch!
(It'd make it even easier for me to fix this if I had a little example code that produces the wrong image when exported.)

**Comment by Duesterwald** [comment]
I am very sorry for neglecting that!

The ouput of the script

```
import matplotlib.pyplot as pp
import matplotlib2tikz as m2t

pp.figure()

pp.plot([1,2,3], [1,2,1], linestyle='--', marker='o')

m2t.save('test.tikz')
```

is

```
% This file was created by matplotlib2tikz v0.5.6.
% The lastest updates can be retrieved from
% 
% https://github.com/nschloe/matplotlib2tikz
% 
% where you can also submit bug reports and leavecomments.
% 
\begin{tikzpicture}

\begin{axis}[
xmin=1, xmax=3,
ymin=1, ymax=2.2,
axis on top
]
\addplot [blue, dashed, mark=*, mark size=3, mark options={draw=black}]
table {%
1 1
2 2
3 1
};
\end{axis}

\end{tikzpicture}
```

It should be something like

```
% This file was created by matplotlib2tikz v0.5.6.
% The lastest updates can be retrieved from
% 
% https://github.com/nschloe/matplotlib2tikz
% 
% where you can also submit bug reports and leavecomments.
% 
\begin{tikzpicture}

\begin{axis}[
xmin=1, xmax=3,
ymin=1, ymax=2.2,
axis on top
]
\addplot [blue, dashed, mark=*, mark size=3, mark options={solid,draw=black}]
table {%
1 1
2 2
3 1
};
\end{axis}

\end{tikzpicture}
```

**Comment by nschloe** [comment]
Fixed by PR #96.
